tests/Jobs/MediawikiInitTest.php

```diff
@@ -76,16 +76,22 @@ public function testSuccess()
     public function testFatalErrorIsHandled()
     {
         $mockResponse = 'oh no';
+        $mockException = new \RuntimeException('wbstackInit call for some.domain.com. No wbstackInit key in response: ' . $mockResponse);
 
         $request = $this->getMockRequest($mockResponse);
 
         $mockJob = $this->createMock(Job::class);
-        $mockJob->expects($this->once())
-                ->method('fail')
-                ->with(new \RuntimeException('wbstackInit call for some.domain.com. No wbstackInit key in response: ' . $mockResponse ));
 
-        $job = new MediawikiInit( $this->wikiDomain, $this->username, $this->email );
-        $job->setJob($mockJob);
-        $job->handle($request);
+        $caughtException = null;
```

Comment:
This test works as is, but feels not right to me. Earlier I tried something like `$mockJob->method('handle')->willThrowException ...` but this didn't work out for me: 
```
1) Tests\Jobs\MediawikiInitTest::testFatalErrorIsHandled
PHPUnit\Framework\MockObject\MethodCannotBeConfiguredException: Trying to configure method "handle" which cannot be configured because it does not exist, has not been specified, is final, or is static
```

Reply:
Without having tried it, I think you can use `expectException`: https://phpunit.de/manual/6.5/en/writing-tests-for-phpunit.html#writing-tests-for-phpunit.exceptions.examples.ExceptionTest.php